README.md

```diff
@@ -29,12 +29,13 @@ docker run --rm -ti --name geth -p 8545:8545 ethereum/client-go:v1.10.26 \
 ### Usage: 
 1. run `yarn && yarn preprocess`
 2. deploy contracts with `yarn hardhat-deploy --network localhost`
-3. run `yarn run bundler`
+3. update `ENTRY_POINT` variable at `packages/bundler/src/runner/runop.ts` to address of the deployed EntryPoint contract 
```

Comment:
fixed the entrypoint address in the script.
anyway, `runop` also parameter a `--entryPoint` parameter, which can be mentioned here instead.

Reply:
Added the new changes, please review again